tools/check-requirements.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+CHECK_RELEASE=${CHECK_RELEASE:-"master"}
+
+for repo in "openstack-ironic" "openstack-ironic-inspector"
+do
+  line=$(grep "$repo@" requirements.cachito)
+  echo $line
+  repo_full=$(echo $line | cut -d "+" -f 2)
+  echo $repo_full
+  commit_hash=$(echo $repo_full | cut -d "@" -f 2)
+  echo $commit_hash
+  git_url=$(echo $repo_full | cut -d "@" -f 1)
+  echo $git_url
+  git clone $git_url
```

Comment:
I'd rather use a temporary directory. Will it fail if this tool is used the 2nd time, and the previous one failed?

Reply:
it won't fail, the repo is removed later in the script
but we can always change this to clone inside a temp dir in a follow up